src/progress.rs

```diff
@@ -233,7 +233,7 @@ impl ProgressSet {
 }
 
 /// The progress of catching up from a restart.
-#[derive(Debug, Default, Clone)]
+#[derive(Debug, Clone)]
```

Comment:
Does a `Progress` have a sane defautt?

Reply:
Yes. An explicit capacity needs to be specified to avoid easily wrong configuration.

Reply:
Ok :+1: